databroker/experimental/example_array.py

```diff
@@ -0,0 +1,25 @@
+import numpy
```

Comment:
this should probably be either deleted or moved into tests

Reply:
Agreed. I think this effectively has been translated into a unit test (elsewhere in this PR) and a notebook (which should go into public-demos) so this copy can be deleted.